Sources/Core/UI/PreviewHelper.swift

```diff
@@ -39,7 +39,7 @@ internal class PreviewHelper {
             return .failure(BoxPreviewError(message: .fileCouldNotBeDownloaded))
         }
         
-        switch unwrappedFileURL.pathExtension {
+        switch unwrappedFileURL.pathExtension.lowercased() {
```

Comment:
We should change also `BMPf` to` bmpf` in line 57. Otherwise this file extension will not be recognized.